projects/contributor-experience.md

```diff
@@ -0,0 +1,118 @@
+# Contributor Experience
+
+## Background and description
+
+Over the last few years the OpenTelemetry project has seen massive growth, not
+only in adoption and lines of code, but also in the number of individuals who
+contribute to this project in different ways.
+
+While in the beginning of the project contributors knew each other well and
+could share implicit in-group knowledge effortlessly, the project has outgrown that
+phase a while ago and we see different kinds of problems, that make it hard
+for long-term contributors to maintain the project sustainably and for new-comers
+to enter the project effortlessly.
+
+In this project proposal we suggest to form a cross-cutting special interest group
+that takes responsibility for driving contributor experience projects and that
+closely collaborates with all other SIGs and governing bodies to increase contributor
+sustainability, velocity and happiness.
+
+### Current challenges
+
+Among others the challenges the OpenTelemetry project is facing in regards of
+contributor experience are:
+
+* No consistent way of moving contributors "up the ladder", e.g. from member to
+  approver, from approver to maintainer etc, which makes those changes invisible
+  to the larger project and opaque to project leadership on who is doing
+  what.
+* No clear onboarding path for new contributors, or contributors taking on new
+  responsibilities within the project (triagers, approvers, maintainers, committee
+  members), which leads to friction in understanding of the overall project goals,
+  to misalignment in implementations and the feeling of contributors being remote
+  to other parts of the project.
+* No explicit contributor guidebook that helps contributors to answer common
+  questions no matter where they are on the contributors' ladder.
+* No mentorship available for new contributors or contributors stepping up into new roles to
+  make them part of the project and aware of expectations.  
+* Only implicit and unstructured recognition for contributors, instead of explicit
+   recognition which would help contributors to feel seen and recognized.
+* No structured collaboration across SIGs, especially in the case of cross-cutting
+  concerns (Security, Docs, End-User, ...), leading to information silos and misalignment.
+* SIGs are short of contributors and require some help to advertise their need
+  (blog post, social posts, etc) and to lower the barrier of entry into their
+  sub-project.
+* _add more_
```

Comment:
This may not be the way to write it, but I'd like to be able to focus on this issue. and also not dismiss it as a problem of the people on that side for not showing up on occasions a secondary timezone meeting was attempted. The root of the issue is we need to refocus on building with conscious and active involvement in making other timezones work. We don't want to accidentally treat one side of the earth as folks who are relegated to watching the others lead, or force them to interrupt their sleep or family duties.

```suggestion
* implicit bias towards founding members timezones, such the majority of SIG meetings occurring at 11pm or midnight UTC+8.
```

Reply:
Thanks for adding this, as someone who is semi-affected by that as well, I fully agree. A layer that adds to the problem is that an answer people often get is "if you want your issue to make progress, please come to the SIG meeting". I understand why we say that and I said (and say) the same thing myself. In a broader sense, there is also an implicit bias that people share a work culture, e.g. they know periods of time where most people are off (e.g. last and first week of the year), they understand certain terminology and to be fluent enough in English to participate in a SIG meeting.

I tried to broaden your suggestion to include that:

```suggestion
* Implicit bias towards US (and EU) working hours, rituals and modes of operation. Majority of SIG meetings occurring from early PST onward.
```

Reply:
thanks for improving my suggestion @svrnm

Reply:
here's the related issue to this for those not following  https://github.com/open-telemetry/community/issues/2176